tensorrt_llm/_torch/pyexecutor/py_executor.py

```
@@ -2639,6 +2644,88 @@ def _sync_and_process_resource_governor_queue(self):
+        torch.cuda.current_stream().synchronize()
+        self._consume_previous_batch_for_rebalance()
+
+        paused: List[LlmRequest] = []
+        for req in self.active_requests:
+            if mgr.is_request_active(req.py_request_id):
+                mgr.suspend_request(req)
+                paused.append(req)
+
+        try:
+            mgr.impl.adjust()
+        except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)